app.py

```diff
@@ -11,7 +11,9 @@ def main():
         real_total = swaScraper.scrape(args)
 
     # If the user doesn't want text notifications, just print the result to the console.
-    if args.no_text:
+    if real_total is None:
+        exit
+    elif args.no_text:
         from datetime import datetime
 
         print("[%s] Found a deal. Max Total: $%s. Current Total: $%s." % (
```

Comment:
Dollar sign should be removed because it can either by $ or points.

Reply:
Oops, missed this one and in notifier.py.  I'm definitely open to a way to specify dollars or points if you have one, but it got too cumbersome because "$" goes before the string and "points" goes after

Reply:
I can write a IF statement for $ or points, don't worry about it.